### DISCUSSION TITLE
crash error: 7fffe4c0739c

### DISCUSSION THREAD

**Discussion by Social-Mean** [texstudio-org/texstudio#1904] [Q&A] [answered]
Today when I opened TeXStudio, it crashed. And its error code is 7fffe4c0739.
And the content of 'texstudio_backtracw1.txt' is 
unknown at 7fffe4c0739c (TXS-Version 4.0.2 R )
[bt] 00007ff67252c360  ??? error: 487
[bt] 000000ac9e9fecb0  ??? error: 126
[bt] 000001e4015d0bd0  ??? error: 126
[texstudio_backtrace1.txt](https://github.com/texstudio-org/texstudio/files/7446064/texstudio_backtrace1.txt)
What should I do now?

**Comment by sunderme** [comment, marked as answer]
can you try the current dev build: https://github.com/texstudio-org/texstudio/actions/runs/1401786222 (artifacts at the bottom of the page)

**Reply by Social-Mean** [reply]
Thanks for your reply.
I have tried the current dev build, but it still not work. 
And now the error code is '63d145'
![image](https://user-images.githubusercontent.com/84260049/139529431-89217ec6-8203-46d5-b163-a1388fb85cd4.png)
(Translation of the image: TeXstudio crash for an unknown reason at 63d145...)

**Reply by Social-Mean** [reply]
I have solved this problem.
I download 'texstudio-win-qt6-exe', and it works.
Thank you!
